### ISSUE TITLE
Pipeline pod failed to start after executing jx install

### ISSUE THREAD

**Issue by IcedSoul** [jenkins-x/jx#6050]
### Summary 
I try to install jx on my local kubernetes cluster, but in the final stage, the pipeline pod never succeeded.
### Steps to reproduce the behavior

```
jx install --provider=kubernetes --git-username=[my github username] --git-api-token=[my github personal token] --domain [my custom domain ]  --on-premise
```
Console output is as follows：

``` 
[root@master ~]# jx install --provider=kubernetes --git-username=[my github username] --git-api-token=[my github personal token] --domain [my custom domain] --on-premise
? Select Jenkins installation type: Serverless Jenkins X Pipelines with Tekton
Context "kubernetes-admin@kubernetes" modified.
Git configured for user: [my github username]  and email [my github email] 
Helm installed and configured
? No existing ingress controller found in the kube-system namespace, installing one: Yes
? A local Jenkins X versions repository already exists, pulling the latest: Yes
? A local Jenkins X versions repository already exists, pulling the latest: Yes

Waiting for external loadbalancer to be created and update the nginx-ingress-controller service in kube-system namespace
Using external IP: apiserver.master
The Ingress address apiserver.master is not an IP address. We recommend we try resolve it to a public IP address and use that for the domain to access services externally.
? Would you like wait and resolve this address to an IP address and use it for the domain? No

Still not managed to resolve address apiserver.master into an IP address. Please try figure out the domain by hand
You can now configure your wildcard DNS icedsoul.cn to point to apiserver.master
nginx ingress controller installed and configured
Set up a Git username and API token to be able to perform CI/CD
? Do you wish to use IcedSoul as the local Git user for GitHub server: Yes
Select the CI/CD pipelines Git server and user
? Do you wish to use GitHub as the pipelines Git server: Yes
Setting the pipelines Git server https://github.com and user name [my github username] .
? A local Jenkins X versions repository already exists, pulling the latest: Yes
? A local Jenkins X cloud environments repository already exists, recreating with latest: Yes
Enumerating objects: 1440, done.
Total 1440 (delta 0), reused 0 (delta 0), pack-reused 1440
Setting up prow config into namespace jx
Installing tekton into namespace jx
? A local Jenkins X versions repository already exists, pulling the latest: Yes

Installing Prow into namespace jx
with values file /root/.jx/cloud-environments/env-kubernetes/myvalues.yaml
? A local Jenkins X versions repository already exists, pulling the latest: Yes

? Defaulting workload build pack: Kubernetes Workloads: Automated CI+CD with GitOps Promotion
Setting the team build pack to kubernetes-workloads repo: https://github.com/jenkins-x-buildpacks/jenkins-x-kubernetes.git ref: master
Installing jx into namespace jx
Installing jenkins-x-platform version: 2.0.1574
? A local Jenkins X versions repository already exists, pulling the latest: Yes


WARNING: waiting for install to be ready, if this is the first time then it will take a while to download images
```

### Expected behavior
All pod running and jx install successful.
### Actual behavior
Pod list:
```
kubectl get pods

NAME                                          READY   STATUS             RESTARTS   AGE
crier-66b54759fd-zfc7c                        1/1     Running            0          3m35s
deck-6dcd4ccb9d-smp54                         1/1     Running            0          3m35s
deck-6dcd4ccb9d-vghbq                         1/1     Running            0          3m35s
hook-8449587b46-cfmpx                         1/1     Running            0          3m35s
hook-8449587b46-tq924                         1/1     Running            0          3m35s
horologium-75ff68df6c-cdqmz                   1/1     Running            0          3m35s
jenkins-x-chartmuseum-774f8b95b-s9gt9         1/1     Running            0          3m24s
jenkins-x-controllerbuild-6f4d6f8d4b-2zbz8    1/1     Running            0          3m23s
jenkins-x-controllerrole-5c794d74b-ngldc      1/1     Running            0          3m22s
jenkins-x-docker-registry-dbf6889cd-qtpqk     1/1     Running            0          3m23s
jenkins-x-heapster-78fd4ccdc9-mgps7           2/2     Running            0          3m16s
jenkins-x-nexus-b87fb7f7b-vf922               1/1     Running            0          3m23s
pipeline-7584b7dd8d-b9k52                     0/1     CrashLoopBackOff   5          3m35s
pipelinerunner-75694b9477-5kv7s               1/1     Running            0          3m35s
plank-5d4d78758c-5h6ld                        1/1     Running            0          3m35s
sinker-548c9d8fbb-t8n6s                       1/1     Running            0          3m35s
tekton-pipelines-controller-b68b4fdcb-g7x95   1/1     Running            0          3m49s
tekton-pipelines-webhook-6979469df4-6nkkq     1/1     Running            0          3m49s
tide-84c6457765-m8mnb                         1/1     Running            0          3m35s
```
 

```
kubectl logs pipeline-7584b7dd8d-b9k52
flag provided but not defined: -tot-url
Usage of /app/prow/cmd/crier/app.binary:
  -alsologtostderr
    	log to standard error as well as files
  -bash-noop-image string
    	The container image containing bash shell (default "override-with-bash-noop:latest")
  -build-cluster string
    	Path to kube.Cluster YAML file. If empty, uses the local cluster. All clusters are used as build clusters. Cannot be combined with --kubeconfig.
  -build-gcs-fetcher-image string
    	The container image containing our GCS fetcher binary. (default "gcr.io/cloud-builders/gcs-fetcher:latest")
  -build-gcs-uploader-image string
    	The container image containing our GCS uploader binary. (default "gcr.io/cloud-builders/gcs-uploader:latest")
  -config-path string
    	Path to config.yaml.
  -cookiefile string
    	Path to git http.cookiefile, leave empty for anonymous
  -deck-url string
    	Deck URI for read-only access to the infrastructure cluster.
  -dry-run
    	Run in dry-run mode, not doing actual report (effective for github only)
  -gerrit-projects value
    	Set of gerrit repos to monitor on a host example: --gerrit-host=https://android.googlesource.com=platform/build,toolchain/llvm, repeat flag for each host
  -gerrit-workers int
    	Number of gerrit report workers (0 means disabled)
  -git-image string
    	The container image containing our Git binary. (default "override-with-git:latest")
  -github-endpoint value
    	GitHub's API endpoint (may differ for enterprise). (default https://api.github.com)
  -github-token-file string
    	DEPRECATED: use -github-token-path instead.  -github-token-file may be removed anytime after 2019-01-01.
  -github-token-path string
    	Path to the file containing the GitHub OAuth secret. (default "/etc/github/oauth")
  -github-workers int
    	Number of github report workers (0 means disabled)
  -gsutil-image string
    	The container image containing gsutil (default "override-with-gsutil-image:latest")
  -job-config-path string
    	Path to prow job configs.
  -kubeconfig string
    	Path to .kube/config file. If empty, uses the local cluster. All contexts other than the default or whichever is passed to --context are used as build clusters. . Cannot be combined with --build-cluster.
  -kubeconfig-writer-image string
    	The container image containing our kubeconfig writer binary. (default "override-with-kubeconfig-writer:latest")
  -log_backtrace_at value
    	when logging hits line file:N, emit a stack trace
  -log_dir string
    	If non-empty, write log files in this directory
  -logtostderr
    	log to standard error instead of files
  -pubsub-workers int
    	Number of pubsub report workers (0 means disabled)
  -report-agent string
    	Only report specified agent - empty means report to all agents (effective for github only)
  -stderrthreshold value
    	logs at or above this threshold go to stderr
  -v value
    	log level for V logs
  -vmodule value
    	comma-separated list of pattern=N settings for file-filtered logging
```
### Jx version

The output of `jx version` is:

```
NAME               VERSION
jx                 2.0.958
jenkins x platform 2.0.1574
Kubernetes cluster v1.16.0
kubectl            v1.16.0
helm client        Client: v2.16.0+ge13bc94
git                1.8.3.1
Operating System   CentOS Linux release 7.7.1908 (Core)
```

### Jenkins type

- [ Yes ] Serverless Jenkins X Pipelines (Tekton + Prow)
- [ ] Classic Jenkins

### Kubernetes cluster
I create this kubernetes cluster with kubeadm.
```
kubectl get nodes
NAME     STATUS   ROLES    AGE   VERSION
master   Ready    master   30d   v1.16.0
node1    Ready    <none>   30d   v1.16.0
node2    Ready    <none>   29d   v1.16.0
node3    Ready    <none>   29d   v1.16.0
```

### Operating system / Environment
```
cat /etc/redhat-release 
CentOS Linux release 7.7.1908 (Core)
```

**Comment by abayer** [comment]
This is weird - somehow it’s using the crier image instead of the pipeline image. @ccojocar, have you ever seen anything like this?

**Comment by IcedSoul** [comment]
> This is weird - somehow it’s using the crier image instead of the pipeline image. @ccojocar, have you ever seen anything like this?

Thank you very much for your reply.
The image I used was first pulled from the DockerHub to the local, and then modified to the specified label: for some well-known reasons, there is no way to pull the image directly from gcr.io in my country. After reading your answer, I checked the command that I modified the label and found that I mistakenly labeled the crier into a pipeline. After repairing the label, the pipeline successfully became running.I am very sorry to open this issue because of personal mistakes.
Thanks again for your reply.
